components/02-elements/button/Button.vue

```diff
@@ -69,7 +69,7 @@ export default {
 }
 </script>
 
-<style lang="scss">
+<style lang="scss" scoped>
```

Comment:
We probably can't do this, b/c styles should be easy to overwrite, there is a high chance, that we will ship it as separate files.

Reply:
It might make sense to not bind these styles to tags, limit it just to classes, will be easier to create custom buttons.